### PR TITLE
docs(clients): Add Cloud client example for js-http

### PR DESCRIPTION
**Pull request by bucanero**
Fixes DOC-80

### CODE REVIEW

File: content/clients/javascript/http.md [hunk 1]
```diff
@@ -56,6 +56,24 @@ const dgraphClient = new dgraph.DgraphClient(clientStub);
 
 To facilitate debugging, [debug mode](#debug-mode) can be enabled for a client.
 
+### Create a Client for Dgraph Cloud Endpoint
+
+If you want to connect to Dgraph running on your [Dgraph Cloud](https://slash.dgraph.io) instance, then all you need is the URL of your Dgraph Cloud endpoint and the API key. You can get a client using them as follows:
```
Comment (OmarAyo) [inline content/clients/javascript/http.md:61]: the link for Dgraph Cloud is wrong, replace it with cloud.dgraph.io instead

File: content/clients/javascript/http.md [hunk 2]
```diff
@@ -56,6 +56,24 @@ const dgraphClient = new dgraph.DgraphClient(clientStub);
...
+
+```js
+const dgraph = require("dgraph-js-http");
+
+//here we pass the slash endpoint
```
Comment (OmarAyo) [inline content/clients/javascript/http.md:66]: replace the `slash endpoint` with `cloud endpoint`

File: content/clients/javascript/http.md (hunk 1 above)
Comment (bucanero) [inline content/clients/javascript/http.md:61]:
```suggestion
If you want to connect to Dgraph running on your [Dgraph Cloud](https://cloud.dgraph.io) instance, then all you need is the URL of your Dgraph Cloud endpoint and the API key. You can get a client using them as follows:
```

File: content/clients/javascript/http.md (hunk 2 above)
Comment (bucanero) [inline content/clients/javascript/http.md:66]:
```suggestion
//here we pass the cloud endpoint
```

File: content/clients/javascript/http.md (hunk 1 above)
Comment (bucanero) [reply content/clients/javascript/http.md]: fixed

File: content/clients/javascript/http.md (hunk 2 above)
Comment (bucanero) [reply content/clients/javascript/http.md]: fixed

File: content/clients/javascript/http.md [hunk 3]
```diff
@@ -56,6 +56,24 @@ const dgraphClient = new dgraph.DgraphClient(clientStub);
...
+const clientStub = new dgraph.DgraphClientStub(
+    "https://super-pail.us-west-2.aws.cloud.dgraph.io",
```
Comment (OmarAyo) [inline content/clients/javascript/http.md:68]: you may want to clarify that they need to remove the `/graphql' endpoint if they copy directelly their endpoint from the dashboar. 

As when copying it from the dashboard the endpoint will be: `https://super-pail.us-west-2.aws.cloud.dgraph.io/graphql` but here the need to remove `/graphql` at the end - if this is not done they will get the following error from node:
```
(node:31883) UnhandledPromiseRejectionWarning: HTTPError: Invalid status code = 404
```

**Review by OmarAyo — CHANGES_REQUESTED** [review]
please address changes

File: content/clients/javascript/http.md (hunk 3 above)
Comment (bucanero) [reply content/clients/javascript/http.md:68]: fixed

**Pull request merged**